sdk/eventhub/azure-eventhub/azure/eventhub/_constants.py

```diff
@@ -45,10 +45,33 @@
 MGMT_STATUS_DESC = b"status-description"
 USER_AGENT_PREFIX = "azsdk-python-eventhubs"
 
-NO_RETRY_ERRORS = (
+NO_RETRY_ERRORS = [
     b"com.microsoft:argument-out-of-range",
     b"com.microsoft:entity-disabled",
     b"com.microsoft:auth-failed",
     b"com.microsoft:precondition-failed",
     b"com.microsoft:argument-error",
-)
+]
+
+CUSTOM_CONDITION_BACKOFF = {
+    b"com.microsoft:server-busy": 4,
+    b"com.microsoft:timeout": 2,
+    b"com.microsoft:operation-cancelled": 0,
+    b"com.microsoft:container-close": 4
+}
+
+
+## all below - previously uamqp
+class TransportType(Enum):
+    """Transport type
+    The underlying transport protocol type:
+     Amqp: AMQP over the default TCP transport protocol, it uses port 5671.
+     AmqpOverWebsocket: Amqp over the Web Sockets transport protocol, it uses
+     port 443.
+    """
+    Amqp = 1
+    AmqpOverWebsocket = 2
+
+DEFAULT_AMQPS_PORT = 5671
+DEFAULT_AMQP_WSS_PORT = 443
+READ_OPERATION = b"READ"
```

Comment:
API change to add TransportType to the SDK level here, rather than calling the one from uamqp.constants.

Reply:
should we wait to make this API change until pyamqp gets merged, and continue to use uamqp TransportType + constants for now?

Reply:
I don't mind much so long as if customers are using the existing enum that they imported directly from uamqp (for some reason), that it would still work.

Reply:
will update one of the tests that takes TransportType to import the one from uamqp, to double check.